src/qt/languages/ru-RU.po

```diff
@@ -2382,8 +2406,26 @@ msgstr "Переключить беззвучный режим"
 msgid "Text files"
 msgstr "Текстовые файлы"
 
+msgid "ROM files (*.bin *.rom)|*.bin,*.rom"
+msgstr ""
+
 msgid "ROM files"
 msgstr "Файлы ПЗУ"
 
 msgid "SoundFont files"
 msgstr "Файлы SoundFont"
+
+#~ msgid "RGB (no brown)"
+#~ msgstr "RGB (без коричневого)"
```

Comment:
You marked this as obsolete, but it's actually used in device configuration for Amstrad PC200, PPC512 and IBM PCjr's internal video.